source/elm-live.js

```diff
@@ -194,9 +194,18 @@ ${indent(String(elmMake.error), 2)}
   };
 
   // Watch Elm files
-  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: true, followSymlinks: false });
+  const watcher = chokidar.watch('.', {
+    ignoreInitial: true,
+    followSymlinks: false,
+    ignored: ['node_modules/', 'elm-stuff/'],
+  });
 
   watcher.on('all', debounce((event, filePath) => {
+    const basename = filePath.substring(filePath.lastIndexOf('/') + 1);
+    // chokidar has got some trouble handling glob pattern with simlink #103
```

Comment:
> I’m trying to avoid the temporary workaround staying around for a long time.

In that specific case the workaround is good enough to stay I guess :)